pkgs/tools/security/pcsc-cyberjack/default.nix

```diff
@@ -1,25 +1,26 @@
-{ stdenv, fetchurl, pkgconfig, libusb1, pcsclite }:
+{ stdenv, lib, fetchurl, pkgconfig, libusb1, pcsclite }:
 
 stdenv.mkDerivation rec {
   pname = "pcsc-cyberjack";
   version = "3.99.5_SP12";
 
   src = with stdenv.lib; let
```

Comment:
Since you already brought `lib` into scope, I think it also makes sense to remove `with stdenv.lib;` everywhere and prefix the corresponding functions with `lib.`.